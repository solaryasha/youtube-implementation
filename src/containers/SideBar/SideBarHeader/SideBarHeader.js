import React from 'react';
import { Menu } from 'semantic-ui-react';
import './SideBarHeader.scss';


const SideBarHeader = (props) => {
    const Header = props.title ? props.title.toUpperCase(): '';

    return (
        <Menu.Item>
            <Menu.Header className='sidebar-header'>{Header}</Menu.Header>
        </Menu.Item>
    );
}

export default SideBarHeader ;