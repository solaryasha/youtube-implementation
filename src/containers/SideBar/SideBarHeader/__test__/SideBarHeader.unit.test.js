import React from 'react';
import {shallow} from 'enzyme';
import SideBarHeader from '../SideBarHeader';

test('Sidebar header rendering', () => {
    const wrapper = shallow(
        <SideBarHeader />
    )
    expect(wrapper).toMatchSnapshot();
})