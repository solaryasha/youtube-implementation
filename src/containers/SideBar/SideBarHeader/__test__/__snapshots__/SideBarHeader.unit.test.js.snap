// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar header rendering 1`] = `
<MenuItem>
  <MenuHeader
    className="sidebar-header"
  />
</MenuItem>
`;
